src/lib.rs

```diff
@@ -59,49 +67,44 @@ pub async fn run_web_server(
 pub async fn run_admin_server(
     http_bind: String,
     state: Arc<RwLock<State>>,
+    loader_rt: Arc<Runtime>,
     close_channel: tokio::sync::oneshot::Receiver<()>,
 ) -> Result<(), Error> {
-    web::admin_server(state, http_bind, close_channel).await
+    web::admin_server(state, http_bind, loader_rt, close_channel).await
 }
 
-pub async fn load_configuration_files(
+pub fn load_configuration_files<'a>(
```

Comment:
lifetime doesn't look useful here

Reply:
It's a leftover from some tests during implementation :sweat_smile: